model/registry/gen-ai.yaml

```diff
@@ -45,6 +45,30 @@ groups:
         brief: The top_p sampling setting for the LLM request.
         examples: [1.0]
         tag: llm-generic-request
+      - id: request.top_k
+        stability: experimental
+        type: double
+        brief: The top_k sampling setting for the LLM request.
+        examples: [1.0]
+        tag: llm-generic-request
```

Comment:
not needed (please remove from other attributes introduced in this PR)
```suggestion
```

Reply:
thanks! done